### PR TITLE
Update README to include information about new `nodejs-backend` template

### PR DESCRIPTION
**Pull request by hbjydev**
See title of PR. This adds a section to the README about the new Node.js Backend template added in #32, as well as some basic (but common) usage information.

### CODE REVIEW

**Comment by hbjydev** [comment]
Requested review mainly to see if you agree with the addition of the extra info on the README about usage :)

File: README.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ Kickstart your Nix environments.
     - [Python (application)](#python-application)
     - [Python (package)](#python-package)
     - [Rust](#rust)
+    - [Node.js (backend)](#nodejs-backend)
```
Comment (erikreinert) [inline README.md:23]: Don't hate me lol let's make sure this is in alphabetical order

File: README.md [hunk 2]
```diff
@@ -246,6 +247,30 @@ Used for Rust applications.
 nix flake init -t github:ALT-F4-LLC/kickstart.nix#rust
 ```
 
+#### Node.js (backend)
```
Comment (erikreinert) [inline README.md:250]: Same for this - not as important but helps the reader know "where" they are if their expectation is that it's always alphabetical.

File: README.md (hunk 1 above)
Comment (hbjydev) [reply README.md:23]: Will do (for both comments)

**Pull request merged**